README.md

```diff
@@ -12,3 +12,7 @@ WinTile can be found on the GNOME Extension site:
 https://extensions.gnome.org/extension/1723/wintile-windows-10-window-tiling-for-gnome/
 
 <img src='demo.gif'>
+
+# Configuration
+1. Visit https://extensions.gnome.org/local/
+1. click on the "Configure this extension" button.
```

Comment:
On Ubuntu 19.10 and up, Chromium is installed as snap by default. As a result, any Chrome extension that use Native Messaging (such as the Gnome extension) does not work. So this settings page is unusable for any user who uses Wintile on Ubuntu (such as myself). Below is a suggestion that includes an alternative approach of configuration:
```suggestion
1. Click on the "Configure this extension" button.
1. Alternatively, open the `Extension` settings in [Gnome Tweaks](https://gitlab.gnome.org/GNOME/gnome-tweaks), locate `Wintile` and click on the cogwheel button to bring up the configuration dialog.
```